pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -2589,7 +2592,7 @@
         "4.5.2",
         "4.5.3"
       ],
-      "BaselineVersion": "4.5.3",
+      "BaselineVersion": "5.0.0",
```

Comment:
@ericstj is it expected this would've been 4.5.3?

Reply:
Yes, you need to undo this.  System.Memory no longer builds in master